vita3k/modules/SceVshBridge/SceVshBridge.cpp

```diff
@@ -806,15 +806,15 @@ EXPORT(int, vshSysconIsDownLoaderMode) {
 }
 
 EXPORT(int, vshSysconIsIduMode) {
-    return UNIMPLEMENTED();
+    return emuenv.cfg.current_config.demo_mode == true ? 1 : 0;
```

Comment:
you can remove == true,  no needed ^^

Reply:
```suggestion
    return emuenv.cfg.current_config.demo_mode  ? 1 : 0;
```
sorry, i have wanted that

Reply:
done